get-subseqs

```diff
@@ -1,3 +1,74 @@
-#!/bin/bash
+#!/usr/bin/python3
 
-python3 src/get-subseqs.py $@
+import sys
+import numpy as np
+
+def get_subseqs():
+
+    try:
+        inFile_1 = sys.argv[1]
+        inFile_2 = sys.argv[2]
+
+        with open(inFile_1,'r') as f1:
+            f1_lines = f1.readlines()
+        with open(inFile_2,'r') as f2:
+            f2_lines = f2.readlines()
+
+        record_list = []
+        header = ''
+        sequence = ''
+        interval = []
+        for line in f1_lines:
+            line = line.strip()
+            if line.startswith('>'):
+                if header != "":
+                    record_list.append([header.strip(), sequence.strip()])
+                    sequence = ""
+                header = line[1:]
+            else:
+                sequence = sequence + line
+        record_list.append([header.strip(), sequence.strip()])
+        record_list = np.array(record_list)
+
+        for line in f2_lines:
+            chrom = line.split()[0]
+            start = int(line.split()[1])
+            end = int(line.split()[2])
+            find_seqs = np.where(np.array(record_list) == chrom)
+            seq = record_list[find_seqs[0]][0][1]
+            print(seq[start-1:end-1])
+            
+    except:
+        inFile_1 = sys.argv[1]
+        inFile_2 = sys.stdin
+
+        with open(inFile_1,'r') as f1:
+            f1_lines = f1.readlines()
+        f2_lines = inFile_2.readlines()
+
+        record_list = []
+        header = ''
+        sequence = ''
+        interval = []
+        for line in f1_lines:
+            line = line.strip()
+            if line.startswith('>'):
+                if header != "":
+                    record_list.append([header.strip(), sequence.strip()])
+                    sequence = ""
+                header = line[1:]
+            else:
+                sequence = sequence + line
+        record_list.append([header.strip(), sequence.strip()])
+        record_list = np.array(record_list)
```

Comment:
Changing the type of what a variable holds, here from a list of lists (which should probably be tuples) is generally a bad idea. Not always, and there is nothing in Python to prevent it (unlike most languages where you wouldn't be able to compile code like this), but it is best avoided. It makes the code much harder to read if you return to it months from now and need to make changes. You need to not only keep track of the state of an object but also what type it currently has and, thus, what you can do with it.